src/griddyn/loads/aggregateLoad.cpp

```diff
@@ -114,15 +114,6 @@ void aggregateLoad::pFlowObjectInitializeA (coreTime time0, std::uint32_t flags)
         // do a first pass of loading
         double rem = 1.0;
 
-        setP (((aggregateLoad *)sLoad)->getP ());  // so this composite load function has direct access to the
-        // zipLoad variables seems a little odd to need to do this but
-        // seems to be a requirement in C++
-        setQ (((aggregateLoad *)sLoad)->getQ ());
-        setIp (((aggregateLoad *)sLoad)->getIp ());
-        setIq (((aggregateLoad *)sLoad)->getIq ());
-        setYp (((aggregateLoad *)sLoad)->getYp ());
-        setYq (((aggregateLoad *)sLoad)->getYq ());
-
```

Comment:
Why was this removed?  I think it is still necessary?

Reply:
Reading it again, you're right. I thought sLoad was `this`, but I'm not sure why. Going to revert this